### ISSUE TITLE
Mieux importer le fichier de traduction

### ISSUE THREAD

**Issue by laem** [cartesapp/cartes#255]
https://github.com/openstreetmap/id-tagging-schema/blob/main/dist/translations/fr.json

Il semble qu'on l'utilise pas correctement dans osmTagLabels.ts. 

Sinon il y a  https://github.com/data-osm/frontend/blob/master/src/assets/i18n/tags-fr.json#L23

**Comment by Kazing** [comment]
En effet j'ai remarqué que les traductions des dernières versions de ID tagging schema, ne sont pas reprise par Cartes

exemple les shop=newsagent sont traduit dans Cartes en "Kiosque à journaux" alors que la dénomination est maintenant "Marchand de journaux"
https://github.com/openstreetmap/id-tagging-schema/commit/defe6cb5564d76b6ed4f80aede3db736185e9bf5#diff-13a3a7df4af9e8d4c2c0c3ef647787d455453dc6e06a0225ee9d75ace729dd48R10379

**Comment by laem** [comment]
Je viens de mettre à jour le paquet NPM https://github.com/cartesapp/cartes/commit/0af0655c2cc7088d1aa636e0d211b9a334a7f31c

Peut-être que ça résoud le pb ?

**Comment by Kazing** [comment]
Oui, c'est parfait, les dernières traductions sont bien affichées

**Comment by laem** [comment]
👍
